packages/harpguru-core/src/components/notification-flash/notification-flash.tsx

```diff
@@ -63,8 +63,13 @@ export const NotificationFlash = ({
           {
             transform: [
               {
-                translateX: translateX,
+                // WARNING: in iOS only; the scale *must* be performed before the
```

Comment:
This comment may be making a false assumption. I'm going to leave it in place as it still offers the caution required but require it is reviewed during this issue: https://github.com/js-jslog/harpguru/issues/87

Reply:
Done: https://github.com/js-jslog/harpguru/issues/87#issuecomment-777741223